feature-libs/checkout/base/core/connectors/checkout/converters.ts

```diff
@@ -0,0 +1,7 @@
+import { InjectionToken } from '@angular/core';
+import { CheckoutState } from '@spartacus/checkout/base/root';
+import { Converter } from '@spartacus/core';
+
+export const CHECKOUT_NORMALIZER = new InjectionToken<
+  Converter<CheckoutState, CheckoutState>
```

Comment:
👍 

more concise to put the actual source instead of any, when we 'normalize' the data coming from the backend. 

However, since we're going with this typing instead of 'any' as usual, maybe it's best to add it in `Occ` as well, so the typing would be `<Occ.CheckoutState, CheckoutState>`

What do you think?

Reply:
vice versa for serializers before we send to the backend